app/models/manageiq/providers/microsoft/inventory/persister/infra_manager.rb

```diff
@@ -0,0 +1,31 @@
+class ManageIQ::Providers::Microsoft::Inventory::Persister::InfraManager < ManageIQ::Providers::Microsoft::Inventory::Persister
+  def initialize_inventory_collections
+    add_infra_collection(:disks)
```

Comment:
just the list of symbols will be probably nicer?

Reply:
something like https://github.com/Ladas/manageiq-providers-amazon/blob/bb1288c74374e09f391cef1a3eed9636052cfac5/app/models/manageiq/providers/amazon/inventory/persister/definitions/cloud_collections.rb#L5